packages/sw-precaching/src/lib/controllers/base-cache-manager.js

```diff
@@ -32,6 +32,28 @@ class BaseCacheManager {
   }
 
   /**
+   *This method gives access to the cache used by thie caching manager.
+   * @return {String} The cache name used for this manager.
+   */
+  getCacheName() {
+    return this._cacheName;
+  }
+
+  /**
+   * This method returns an array of URL's that will be cached by this
```

Comment:
Could you adjust this comment to make it clear that you're returning fully qualified URLs (even though the manifest contains relative URLs)?